convert/README.md

```diff
@@ -0,0 +1,185 @@
+	print-stack         Print CDK stack resources
+
+### Create terraform variables
+
+First, we must create the variables going into our Terraform config.
+
+The following command will look at the current CDK stack and autopropagate the Terraform variables:
+
+    ./convert.py create-tfvars --region us-east-1 --stack-name mystack --ssh-key-path /path/to/key.pem > terraform/terraform.tfvars.json
+
+Note that the ssh key is handled different in the terraform module. With CDK,
+we used an existing keypair, while the terraform module creates a keypair for
+you. The local key does not need to be the one you used with CDK. The new
+autoscaling group nodes and bastion node will be provisioned with the new key.
+
+### Get AWS resources to import
+
+Then, we'll use `convert.py` to generate the Terraform import commands to import our AWS resources:
+
+    ./convert.py get-imports --region us-east-1 --stack-name mystack > terraform/imports.sh
+
+### Initialize and import Terraform resources
+
+Change to the `terraform/` directory, and run the following two commands:
+
+    terraform init
+    bash imports.sh
+
+`imports.sh` will run *several* `terraform import` commands in a row. This is normal.
+
+### Evaluate the Terraform plan
+
+Once everything has been imported, you can run `terraform plan` and evaluate what terraform will do:
+
+    terraform plan -out terraform.plan
+
+There will be a lot of output, but near the end you should see output similar to this:
+
+    Plan: 111 to add, 34 to change, 0 to destroy.
+
+If there is anything Terraform plans to destroy, that is *not* expected and you should carefully scrutinize the output to ensure Terraform isn't doing anything destructive.
+
+Regardless, you should peruse the output to understand what is going to happen. It should be creating new node groups, various security groups and IAM policies, a new bastion, and otherwise leaving things unchanged save for minor tag changes, etc. Major red flags would be large changes or delete/create cycles of the EKS cluster itself, S3 buckets, or EFS.
+
+### Apply the terraform plan
+
+If you're satisfied that the `terraform plan` output is reasonable, you can run `terraform apply`:
+
+    terraform apply terraform.plan
+
+Once Terraform is finished, the most critical work is done.
+
+### Scale down the old autoscaling groups
+
+The Domino UI will not be immediately accessible, as the old auto-scaling groups will no longer have all the required permissions for proper functionality. You will have two sets of autoscaling groups at this point, so you will want to scale down the old ones.
+
+Set min/max/desired to 0 on all CDK-provisioned auto-scaling groups.
+
+The *old* ones look like this:
+
+    clustername-platform-0-us-east-1
+
+The *new* ones look like this:
+
+    eks-clustername-platform-0-abcd1234-abcd-1234-5678-abcd12345678
+
+Once all the Domino pods are live on the *new* nodes, Domino should be back up and functional.
```

Comment:
Part if what's shown in the readme is out of date. I made it so the ASGs actually work right after provision. I'll update that top part of the phrasing.

In a follow up thing, I'm adding the ability to delete the ASGs. This will be without a kubectl cordon/drain, so ungraceful. My plan is to make it so this can still be semi-automated via clean-stack doing this dumber procedure. I'd prefer to avoid automating anything on the kubectl level other than what's already done already in the terraform-aws-eks.

Reply:
Updated the top part to be accurate. Since I'll be working on the ASGs anyway, if we do want to finesse it beyond that can address in follow up (and work with a smaller changeset...).